tutorials/run-swift-vapor-on-google-cloud/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ibmcom/swift-ubuntu:latest
+LABEL Description="Docker image for Swift + Vapor on GAE Flex."
+
+# Expose default port for GAE
+EXPOSE 8080
+
+# Copy sources
+RUN mkdir /root/VaporGAE
+ADD main.swift /root/VaporGAE
+ADD Package.swift /root/VaporGAE
+
+# Build the app
+RUN cd /root/VaporGAE && swift build
```

Comment:
Better to set`WORKDIR /app` here, then you can just have a `RUN swift build` step and a simpler `CMD` below.